index.js

```diff
@@ -153,54 +153,87 @@ function SpotifyWebHelper(opts) {
 			});
 		});
 	};
-	this.generateSpotifyUrl = function (url) {
+	this.generateSpotifyUrl = function (url, port) {
+		port = typeof port === "undefined" ? localPort : port;
```

Comment:
This works, but stylistically `port || localPort` makes more sense.

Reply:
Ahh, im still too used to java tbh xD I forget that that is possible

Reply:
wouldn't for node.js function (url, port = localPort) actually be better as its emca 5 in node.js?